releng/lib/check_artifacts.py

```diff
@@ -202,12 +212,24 @@ def do_check_docker(checker: Checker, name: str) -> None:
         if release_channel != '':
             check_tag = f"{check_tag}-{release_channel}"
         assert_eq(check.result, check_tag)
+
     with checker.check(name='ambassador.git GitHub release for chart') as check:
-        check.result = 'TODO'
-        raise NotImplementedError()
+        tag = run_txtcapture([
+            "gh", "release", "view",
```

Comment:
Like I said above, let's add a comment here about how this doubles as checking the tag.

Reply:
Done.